exercise_20.js

```diff
@@ -0,0 +1,20 @@
+/* Create a function which answers the question "Are you playing banjo?".
+If your name starts with the letter "R" or lower case "r", you are playing banjo!
+
+The function takes a name as its only argument, and returns one of the following strings:
+
+name + " plays banjo"
+name + " does not play banjo"
+Names given are always valid strings.
+ */
+
+function areYouPlayingBanjo(name) {
+    if (name[0] === "r" || name[0] === "R") {
+        return name +  ` plays banjo`;
+    } else {
+        return name + ` does not play banjo`;
+    }
```

Comment:
typically when we use backtics then we write it like:
```
return `${name} plays banjo`;
```
and when we do not then:
```
return name +  " plays banjo";
or
return name +  ' plays banjo';
```